### PR TITLE
chore(test): add a long running test for scaling up and down replicas

### PR DESCRIPTION
**Pull request by dsharma-dc**

### CODE REVIEW

**Comment by dsharma-dc** [comment]
@tiagolobocastro  Please take a look. The test runs ok but the last call to delete volume is failing with connection error.

**Comment by tiagolobocastro** [comment]
> @tiagolobocastro Please take a look. The test runs ok but the last call to delete volume is failing with connection error.

Would need to check the logs...

File: tests/io-engine/tests/rebuild.rs [hunk 1]
```diff
@@ -986,3 +986,128 @@ async fn destroy_rebuilding_nexus() {
...
 }
+
+/// This test will create a volume with two replicas at the beginning
+/// and start fio to the volume. Once the fio is running in background,
+/// the replica count will be scaled up and down repeatedly.
+/// NOTE: This test is long running and hence default marked as 'ignore'.
+/// Not meant to be run as part of CI.
+#[tokio::test]
+#[ignore]
+async fn replica_scaling_rebuild() {
```
Comment (tiagolobocastro) [inline tests/io-engine/tests/rebuild.rs:997]: What's the difference between this test and repeated_volume_rebuilds ?

**Comment by dsharma-dc** [comment]
Closing as this is not required anymore and we have a similar test already.

**Pull request closed without merging**